source/init/mus_initFluid_module.f90

```diff
@@ -277,17 +217,136 @@ subroutine mus_init_advRel_fluid( relaxation, layout, compute )
       case ('d2q9')
         compute => bgk_Regularized_d2q9
       case default
-        write(logUnit(1),*) 'Stencil '//trim(layout)//' is not supported yet!'
-        call tem_abort()
+        call tem_abort('Stencil '//trim(layout)//' is not supported yet!')
       end select
 
     case default
-      write(logUnit(1),*) 'Relaxation '//trim(relaxation)//' is not supported!'
-      call tem_abort()
+      call tem_abort('Relaxation '//trim(relaxation)//' is not supported!')
     end select
 
   end subroutine mus_init_advRel_fluid
   ! **************************************************************************** !
 
+  ! ************************************************************************** !
+  !> This routine assigns compute routine for bgk relaxation
```

Comment:
We ought to document the availble Kernels, that the user may select in the configuration somewhere.

Reply:
documented supported variants for relaxation bgk, mrt and trt